src/components/csvIntegration.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import { Segment, Button, Icon } from "semantic-ui-react";
+import { upperFirst } from "lodash"
+
+import { urlCsvIntegrtation } from '../urls';
+
+import style from "../styles.css";
+
+export class CsvIntegration extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modelName: upperFirst(props.componentName)
+    };
+  }
+
+  componentDidMount() {
+    document.addEventListener("keydown", this.handleKeyPress, false);
+  }
+  componentWillUnmount() {
+    document.removeEventListener("keydown", this.handleKeyPress, false);
+  }
+
+  handleKeyPress = e => {
+    if (e.keyCode === 27) {
+      this.props.handleCsvHide(this.props.componentName);
+    }
+  };
+
+  render() {
+    const { componentName, appDetails } = this.props;
+    let heading = this.state.modelName;
+    return (
+      <Segment basic>
+        <Segment attached="top" styleName="style.headingBox">
+          <h3 styleName="style.heading">Add {heading}s via CSV</h3>
+          <Icon
+            color="grey"
+            name="delete"
+            onClick={() => this.props.handleCsvHide(componentName)}
+          />
+        </Segment>
+        <Segment style={{ width: "30vw" }} attached>
+        	<Button
+            color={appDetails.theme}
+            href={`${urlCsvIntegrtation()}?model=${this.state.modelName}`}
```

Comment:
@pradumangoyal told a better way for this. You can use `axios.get` to get the file content and download it using the JavaScript [File](https://developer.mozilla.org/en-US/docs/Web/API/File) class in one click.

Reply:
You may refer this https://github.com/IMGIITRoorkee/omniport-frontend-no-dues/blob/62bffbd1f613e2a88ab4d7ca71437587d0fa8639/src/components/Verifier/home.js#L294